airflow/www/static/js/dag/details/taskInstance/BackToTaskSummary.tsx

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { Button, Flex } from '@chakra-ui/react';
+
+interface Props {
+  isMapIndexDefined: boolean;
+  onClick: () => void;
+}
+
+const BackToTaskSummary = ({ isMapIndexDefined, onClick }: Props) => {
+  if (!isMapIndexDefined) return null;
+
+  return (
+    <div>
```

Comment:
Looks like the `<div>` got added back again.

Reply:
removed